### DISCUSSION TITLE
Hap-Homematic, Addon bleibt hängen

### DISCUSSION THREAD

**Discussion by nucomagpernozzoli** [thkl/hap-homematic#638] [General]
Hallo Zusammen

Ich habe ein Problem mit dem Homematic Hap Addon. In Betrieb ist eine CCU 3 Zentrale auf einem asus thinker board s. 
Aktuelle Software ist: 3.69.7.20230626.

Seit dem neuen Update und auch schon vorhin stockt das System extrem. Auch die Rückmeldungen kommen nicht sauber durch.

Habt Ihr Erfahrung mit dem? 
![Hap-Homematic_Addon](https://github.com/thkl/hap-homematic/assets/71671015/07ddffc3-07db-4f3c-b58b-b0a75eca656b)

Die Ports freigeschaltet und die HTTP Weiterleitung sind deaktiviert.
[](url)

**Comment by blackbasket** [comment]
Hey... mit Asus hab ich keine Erfahrung... ich hab meine CCU3 heute auf das aktuellste RM-Build gehoben und keine Probleme... siehst du denn irgendwas in den Logs?
